packages/fxa-auth-client/lib/recoveryKey.ts

```diff
@@ -12,6 +16,30 @@ async function randomKey() {
   return recoveryKey;
 }
 
+async function getRecoveryKeyId(
```

Comment:
nit: Seems like this and function below could be combined into one?

Reply:
We're using this now-extracted `getRecoveryKeyId` in both `getRecoveryKeyIdByUid` (function below) and `generateRecoveryKey`.